### PR TITLE
update profiler

### PR DESCRIPTION
**Pull request by pfitzseb**

### CODE REVIEW

File: styles/julia-client.less [hunk 1]
```diff
@@ -97,6 +97,25 @@ atom-text-editor.editor {
...
+        background: fade(@background-color-warning, 20%);
+      }
+    }
+    &.isgc > div {
+      border: solid fade(@background-color-error, 20%) 1px;
+      background: fade(@background-color-error, 10%);
+      &:hover {
+        background: fade(@background-color-error, 20%);
+      }
+    }
```
Comment (aviatesk) [inline styles/julia-client.less]: I tried to swap these colourings thinking about the consistency with ProfileView, but just found the profile view full of red colours is less cool haha:
image

Comment (pfitzseb) [reply styles/julia-client.less]: Hm, ideally we'd have a legend to explain what the colors mean (or maybe show "gc"/"dynamic dispatch" next to the function name).

Comment (aviatesk) [reply styles/julia-client.less]: I don't know which is better, but a legend will occupy some space in profiler view (which currently looks beautiful and cool) and will look a bit too much I guess ?

Comment (aviatesk) [reply styles/julia-client.less]: How about this ?
image

Just renaming classes and reuse them for the latter solution

Comment (pfitzseb) [reply styles/julia-client.less]: Looks good to me. Would be nice to have all text under the toolbar, if you want to take a look at that.

Comment (aviatesk) [reply styles/julia-client.less]: > Would be nice to have all text under the toolbar

sounds better, will push a commit later with the final image.

Comment (aviatesk) [reply styles/julia-client.less]: hm after taking some time to find a way to separate those texts into an new `span`, 
I found we need to tweak the `ToolBar` ~ `ProfileViewer` chain a bit and it's somewhat not just easy -- I think we need some additional logic to accept optional texts under the toolbar here:
https://github.com/JunoLab/atom-ink/blob/7803ad2ce517a4e985d945de24af7e81b6b7143d/lib/plots/pane.js#L60
but still not figured out how exactly I should do.

I can tackle this again when I have time (maybe weekend I guess, I'm currently rather busy for my work), but do you want me to push the changes that at least implements the above image for now ?

Comment (pfitzseb) [reply styles/julia-client.less]: Yeah, go for it. We can move the text around in a follow-up PR.

Comment (aviatesk) [reply styles/julia-client.less]: okay !

**Pull request merged**